doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc

```diff
@@ -0,0 +1,48 @@
+[id='secret-adminuser-create-proc']
+:INCLUDECENTRAL!:
+ifeval::["{context}"=="openshift-authoring"]
+:INCLUDECENTRAL: {CENTRAL}
+endif::[]
+ifeval::["{context}"=="openshift-authoring-managed"]
+:INCLUDECENTRAL: {CENTRAL}
+endif::[]
+ifeval::["{context}"=="openshift-freeform"]
+:INCLUDECENTRAL: {CENTRAL} Monitoring
+endif::[]
+ifeval::["{context}"=="openshift-managed"]
+:INCLUDECENTRAL: {CENTRAL} Monitoring
+endif::[]
+//immutable handled separately, so do not define INCLUDECENTRAL for it
+
+
+= Creating the secret for the administrative user
+
+You must create a secret that contains the user name and password for a {PRODUCT} administrative user account. This secret is required for deploying {PRODUCT} using any template except the trial template. 
+
+If you are using multiple templates to deploy components of {PRODUCT}, use the same secret for all these deployments. The components utilize this user account to communicate with each other.
+
+ifdef::INCLUDECENTRAL[]
+You can also use this user account to log in to {INCLUDECENTRAL}. If you do not use RH-SSO or LDAP authentication, use this account to create other user accounts.
```

Comment:
I don't understand this sentence.
If you do not use RH-SSO or LDAP, the user won't be able to create any more user accounts.
If you do use RH-SSO or LDAP, these credentials will be used by KIE Server and Business/Decision Central to communicate with each other and must exist in RH-SSO or LDAP with the same exact values as it is mentioned in the next sections.